ci-operator/step-registry/ipi/conf/debug/kdump-configure-logs/ipi-conf-debug-kdump-configure-logs-commands.sh

```diff
@@ -7,58 +7,62 @@ set -o pipefail
 node_role=${APPLY_NODE_ROLE:=worker}
 log_path=${LOG_PATH:="/var/crash"}
 
-echo "Creating kdump configuration"
-kdump_conf=$(cat <<EOF | base64 -w 0
-path $log_path
-core_collector makedumpfile -l --message-level 7 -d 31
-EOF
-)
+openshift_release="4.11.0"
+if [[ $(echo $RELEASE_IMAGE_LATEST | grep -o -e "4\.[0-9]\+\.") ]]; then
+  openshift_release="$(echo $RELEASE_IMAGE_LATEST | grep -o -e '4\.[0-9]\+\.').0"
+fi
 
-echo "Creating kdump sysconfig"
-kdump_sysconfig=$(cat <<EOF
-KDUMP_COMMANDLINE_REMOVE="${KDUMP_COMMANDLINE_REMOVE}"
-KDUMP_COMMANDLINE_APPEND="${KDUMP_COMMANDLINE_APPEND}"
-KEXEC_ARGS="${KDUMP_KEXEC_ARGS}"
-KDUMP_IMG="${KDUMP_IMG}"
-EOF
-)
-echo "$kdump_sysconfig"
-base64_kdump_sysconfig=$(echo "$kdump_sysconfig" | base64 -w 0)
+echo "Crash kernel set to ${CRASH_KERNEL_MEMORY}"
 
 echo "Configuring kernel dumps on $node_role nodes"
-cat >> "${SHARED_DIR}/manifest_99_kdump_machineconfig.yml" << EOF
-apiVersion: machineconfiguration.openshift.io/v1
-kind: MachineConfig
+cat >> "${SHARED_DIR}/99_${node_role}_kdump.bu" << EOF
```

Comment:
This needs to be changed to be:
manifest_... scan that specific string is what is looked for to pull in the info.